app/views/katello/api/v2/sync_plans/show.json.rabl

```diff
@@ -2,7 +2,7 @@ object @resource
 
 attributes :id, :organization_id
 attributes :name, :description
-attributes :interval, :next_sync
+attributes :interval, :next_sync, :sync_date_with_offset
```

Comment:
`sync_date_with_offset` makes me think there should be a regular `sync_date` field but I don't see one here?